ajax/libs/brython/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "brython",
+  "filename": "www/src/brython.min.js",
+  "version": "3.2.7",
+  "description": "Python 3 in the browser",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/brython-dev/brython.git"
+  },
+  "keywords": [
+    "Python",
+    "browser"
+  ],
+  "author": "Pierre Quentel",
+  "license": "BSD-2-Clause",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/brython-dev/brython.git",
+    "basePath": "",
+    "files": [
+      "www/src/brython.js",
+      "www/src/brython_dist.js"
```

Comment:
Because the author said that we can just add the two files.

https://github.com/brython-dev/brython/issues/454